### ISSUE TITLE
GetMatches returns inconsistent data

### ISSUE THREAD

**Issue by rhalbheer** [BenFradet/RiotSharp#546]
I am running of a DeveloperInstance. When I call 
`var AllMatches = api.GetMatchList(RiotSharp.Misc.Region.euw, Summoner.AccountId);`
I get the last 100 games of a Summoner, which works fine. However, Season is always on 0 ("PreSeason3") and region is always br. If I run the query on developer.riotgames.com I get the right results (Season 11, mostly and euw in my case).
```

  | Name | Value | Type
-- | -- | -- | --
  | ChampionID | 154 | long
  | GameId | 3610851530 | long
  | Lane | Jungle | RiotSharp.Misc.Lane
  | PlatformID | EUW1 | RiotSharp.Misc.Platform
  | Queue | "440" | string
  | Region | br | RiotSharp.Misc.Region
  | Role | None | RiotSharp.Misc.Role
  | Season | PreSeason3 | RiotSharp.MatchEndpoint.Enums.Season
▶ | Timestamp | {24.04.2018 19:19:21} | System.DateTime

```
This is copy/paste of one object out of the debugger

Is this a bug or did I miss something?
Thank you
Roger

**Comment by BenFradet** [comment]
Yes this is most likely a bug in how the wrapper is deserializing region and season, can you show a side by side between what's returned by `developer.riotgames.com` and the wrapper?

**Comment by rhalbheer** [comment]
Sure, here you go:
`            "lane": "JUNGLE",
            "gameId": 3610851530,
            "champion": 154,
            "platformId": "EUW1",
            "timestamp": 1524597561089,
            "queue": 440,
            "role": "NONE",
            "season": 11`

What I just saw: Season is correct in Riot's answer, it would have to be Season 8, however, region does not show in the JSON response.

Roger

**Comment by BenFradet** [comment]
This is most likely due to the `SeasonConverter` not being used [here](https://github.com/BenFradet/RiotSharp/blob/76d40ec451a5d21a28f8132008acb6b5aaf868c6/RiotSharp/Endpoints/MatchEndpoint/MatchReference.cs#L57-L58), feel free to open a PR.
